### ISSUE TITLE
mono .paket/paket.exe generate-include-scripts failed on OSX

### ISSUE THREAD

**Issue by mastoj** [ionide/ionide-vscode-paket#33]
Tried to run `Paket: Generate Include Sripts' from inside VS Code and it failed with the message:

    command 'paket.GenerateIncludeScripts' not found

VS Code version: 1.9.0
Ionide Paket version: 1.6.0
Ionide Fake version: 1.2.3
Ionide FSharp verison: 2.22.0
Mono version: 4.8
OSX Version: 10.12.13

My paket.dependencies had the following content:

```
source https://nuget.org/api/v2

nuget FSharp.Data framework: net45
nuget MySql.Data framework: net45
nuget Newtonsoft.Json framework: net45
```

I didn't have a fsproj since I was only playing around with a script file.

**Comment by smoothdeveloper** [comment]
@mastoj, could you try to reproduce from command line by running

 > mono .paket/paket.exe generate-include-scripts --verbose

This should give more details

**Comment by mastoj** [comment]
@smoothdeveloper, not sure what is going on, but all of sudden everything works as expected. It always worked from the command line, it was only failing from inside code.

I think we can close it since I can't reproduce it anymore.

**Comment by Krzysztof-Cieslak** [comment]
Looks like `paket.GenerateIncludeScripts` is not added as activation event in package manifest. Which means that it will return 
```
command 'paket.GenerateIncludeScripts' not found
```
if user hasn't opened any paket file, or hasn't run any other paket command.

To reproduce problem - open fresh instance of Code, and try to run `Paket: Generate Include Sripts` before doing anything else.

**Comment by Krzysztof-Cieslak** [comment]
Should be fixed in `1.6.1`
